streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -246,10 +246,15 @@ State setState(final State newState) {
 
     public boolean isRunning() {
         synchronized (stateLock) {
-            return state.isAlive();
+            return getIsAlive();
         }
     }
 
+    //For KafkaStreamTest
+    public boolean getIsAlive() {
```

Comment:
this naming is a bit weird to me. How about `isStateAlive`?

Reply:
copy that